galaxy_ng/app/api/ui/serializers/collection.py

```diff
@@ -43,6 +43,24 @@ class CollectionVersionSummarySerializer(Serializer):
     created = serializers.CharField()
 
 
+class CollectionRemoteSerializer(Serializer):
+    name = serializers.CharField()
```

Comment:
You should use a ModelSerializer. That way it will automatically handle some CRUD operations for you and you don't have to define the field types for each field. You should be able to import the [CollectionRemote](https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/app/models.py#L211) model from pulp ansible

Here's an example of a model serializer: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/api/ui/serializers/namespace.py#L19